### DISCUSSION TITLE
maintain fisheye view when zooming / moving to away from Earth

### DISCUSSION THREAD

**Discussion by AlexMitchellMus** [Stellarium/stellarium#2042] [Q&A] [answered]
Hi,

When in Fisheye view, and move to different planets - stars - objects, the fisheye view is not maintained. It looks like we are simply zooming into the object, eg:
![fisheye_setting](https://user-images.githubusercontent.com/12004932/142255863-b15abede-5647-47a7-b96f-6c580e03e920.png)
![saturn_fisheye_stellarium](https://user-images.githubusercontent.com/12004932/142256164-1b221f6d-f0b1-4e8c-a20b-e334925be8a5.png)

When we zoom into Saturn in Celestia it looks like this in fisheye mode:
![Screenshot from 2021-11-18 02-32-57](https://user-images.githubusercontent.com/12004932/142256081-a73fafe7-c926-4fbe-8c25-2b02b48bf7d3.png)

**Comment by gzotti** [comment]
When you zoom in with the projection set in fisheye mode, the projection mathematics will still project like fisheye optics, even though the difference against e.g. perspective view in this "high focal length" setting will be negligible. However, when you want to see Saturn in Stellarium just like in the Celestia view, you must go into the physical vicinity of Saturn. E.g., place yourself on a moon of Saturn.

**Reply by AlexMitchellMus** [reply]
Thanks @gzotti, can you point me to instructions how to place the camera at different places?

Is it possible to fly? Or do we have to move directly to a position?

**Comment by gzotti** [comment, marked as answer]
Select a moon and press Ctrl+G ("Go there"). Other shortcuts and much more is available in the F1 help menu and the User Guide.
